src/rapids_singlecell/preprocessing/_neighbors.py

```diff
@@ -21,7 +21,7 @@
     from anndata import AnnData
 
 AnyRandom = None | int | np.random.RandomState
-_Alogithms = Literal["brute", "ivfflat", "ivfpq", "cagra"]
+_Alogithms = Literal[_Alogithms_bbknn, Literal["nn_descent"]]
```

Comment:
```suggestion
_Algorithms = _Algorithms_bbknn | Literal["nn_descent"]
```

or `Union` if you have to